.github/workflows/ci.yml

```diff
@@ -95,6 +150,14 @@ jobs:
       # SCCACHE_RECACHE: 1 # Uncomment this to clear cache, then comment it back out
     steps:
       - uses: actions/checkout@v2
+
+      # If this job is being run by Bors (it was pushed to staging),
+      # then build and run in release mode
+      - name: Set RELEASE mode
+        if: github.event_name == 'push' && github.ref == 'refs/heads/staging'
+        run: |
+          echo "RELEASE=true" | tee -a $GITHUB_ENV
```

Comment:
Does this value need to persist between jobs? It seems like you check that this var is set from the `k8_cluster_test` job.

You can do that with `::set_output`. Check out the Release workflow for example.

Reply:
Hmm you're right. I think I can just go back to checking `github.event_name == 'push' && github.ref == 'refs/heads/staging'`

I think I was trying to be slick and just check RELEASE because it better describes what we we're really interested in, but it's probably better to not try to be fancy, it'd probably make things more brittle

Good catch by the way, nice eye